vernac/vernacentries.ml

```diff
@@ -1619,13 +1619,12 @@ let vernac_check_may_eval ~atts redexp glopt rc =
   let glopt = query_command_selector ?loc:atts.loc glopt in
   let (sigma, env) = get_current_context_of_args glopt in
   let sigma', c = interp_open_constr env sigma rc in
-  let c = EConstr.Unsafe.to_constr c in
   let sigma' = Evarconv.solve_unif_constraints_with_heuristics env sigma' in
   Evarconv.check_problems_are_solved env sigma';
-  let sigma',nf = Evarutil.nf_evars_and_universes sigma' in
+  let sigma' = Evd.minimize_universes sigma' in
   let uctx = Evd.universe_context_set sigma' in
   let env = Environ.push_context_set uctx (Evarutil.nf_env_evar sigma' env) in
-  let c = nf c in
+  let c = EConstr.to_constr ~abort_on_undefined_evars:false sigma' c in
   let j =
```

Comment:
You should probably seize the opportunity to move the broken `to_constr` call to the `else` branch below, it's an easy cleanup.

Reply:
Done.